git-refresher/pull-requests.qmd

```diff
@@ -0,0 +1,99 @@
+--- 
+title: "Pull requests"
+---
+
+A [pull request](../glossary.html#pull_request)  allows you to notify your team about changes you've committed and pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add any necessary follow-up commits before the changes are merged into the main branch.
+
+Now you have made changes in your [development branch](making-a-development-branch), we will practice this by merging the changes into your main branch using a pull request.
+
+## Creating pull requests
+
+Once you have made all the changes you want to include in this batch of work, you'll bundle all of your commits together into a pull request for review.
+
+::: panel-tabset
+## Git Bash
+
+1. Ensure you are on the development branch. 
+
+``` bash
+git checkout -b <github_username>/development
+```
+
+2. Check all your latest changes are pushed to the remote repository. 
+``` bash
+git push origin <github_username>/development
+```
+3. Create your pull request, adding a title and a description of your changes 
+
+``` bash
+gh pr create --base <github_id>/main --head <github_id>/development --title "Update to log book" --body "Updating logbook to reflect completing the development branch task."
+```
+
+## Rstudio and GitHub Website
+
+1. In the Git panel, ensure you are on the development branch. 
```

Comment:
I think just shift 1 and 2 to a call-out box before the tabset and just have it as something along the lines of "Before attempting to create your pull request, make sure to have pushed any changes on the branches involved to the remote repo on GitHub"